src/parser/parser.yy

```diff
@@ -1438,6 +1438,9 @@ show_sentence
     | KW_SHOW KW_PARTS {
         $$ = new ShowSentence(ShowSentence::ShowType::kShowParts);
     }
+    | KW_SHOW KW_PARTS INTEGER {
```

Comment:
`KW_PARTS` ? or `KW_PART`?

Reply:
Maybe both? It seems most of command in console is `xxxxs`.

Reply:
Aha，A difficult choice 😄

Reply:
both supported for now